src/main/java/com/lenis0012/bukkit/marriage2/commands/CommandHeal.java

```diff
@@ -40,7 +42,10 @@ public void execute() {
         }
 
         // Check health of partner
-        double give = Math.min(partner.getMaxHealth() - partner.getHealth(), player.getHealth() - 0.5);
+        AttributeInstance partnerMH = partner.getAttribute(Attribute.GENERIC_MAX_HEALTH);
```

Comment:
what's the difference between GENERIC_MAX_HEALTH and getMaxHealth() ?

Reply:
Main difference is that getMaxHealth() is deprecated, and the Attribute system has since replaced it's functionality. Again, though... If the goal is to keep 1.8 support, getMaxHealth() should still work.